### PR TITLE
#211 - about company page, update main page

### LINKED ISSUE #211: About company page
Należy zaimplementować stronę z informacjami o firmie Blumilk.
Link: https://xd.adobe.com/view/26f66876-cddd-469c-8b1f-b991dec40e55-73a5/screen/4c3c1552-b894-47d4-8f8a-d9bac482bd54/

Teksty na tej podstronie razcej nie będą często zmieniane, więc można zahardcodować treści.

Zdjęcia w sekcji “po godzinach” - to ewentualnie mogłoby się dodawać z panelu admina, żeby na bieżąco aktualizować. Zastanowić się jednak, jak zachować ten układ

obraz

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
Created about page, but also added simple bubbles animation, change button placement and hover buttons style in nav, added binary img in footer, switched tech images to similarly sized ones.
image

It should close #211

### CODE REVIEW

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. I don't know why, but on my laptop, the font is different form that in design


My laptop:

image


Design:

image


2. When I hover on images, they overlap a bit - please see the video below

https://github.com/blumilksoftware/website/assets/56546832/149b6b36-2d7c-4b7d-9093-05a1078b5560


3. On mobile - too much space above the image

image

4. On mobile I can scroll left-right, but I shouldn't.

**Comment by AleksandraKozubal** [comment]
> 1. I don't know why, but on my laptop, the font is different form that in design
> 
>  2. When I hover on images, they overlap a bit - please see the video below
> 
>  3. On mobile - too much space above the image
> 
>  4. On mobile I can scroll left-right, but I shouldn't.

1. changed
2. We decided to leave it for now, coz it looks ok on my laptop, tested on firefox and edge
3. removed
4. fixed

**Review by EwelinaSkrzypacz — DISMISSED** [review]
I created issue for this overlapping images -> https://github.com/blumilksoftware/website/issues/248

But we will leave for now, so :+1:

File: resources/views/about.blade.php [hunk 1]
```diff
@@ -1,5 +1,172 @@
...
+                                </div>
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/plane.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="h-36 2xl:h-52 3xl:h-72 w-full place-items-center md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                             data-taos-offset="400">
+                            <img src="{{ asset('images/photos/birthday.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-105" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:152]:
```suggestion
                            <img src="{{ asset('images/photos/birthday.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-105" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 2]
```diff
@@ -1,5 +1,172 @@
...
+                                </div>
+                            </div>
+                            <div class="grid grid-rows-2 justify-self-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/meetup.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
+                                </div>
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/plane.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:146]:
```suggestion
                                    <img src="{{ asset('images/photos/plane.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 3]
```diff
@@ -1,5 +1,172 @@
...
+                                </div>
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 delay-300 duration-1000 taos:scale-75 taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/energylandia.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
+                                </div>
+                            </div>
+                            <div class="grid grid-rows-2 justify-self-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/meetup.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:142]:
```suggestion
                                    <img src="{{ asset('images/photos/meetup.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 4]
```diff
@@ -1,5 +1,172 @@
...
+                        <div class="grid grid-cols-3 place-content-evenly gap-2 2xl:gap-6">
+                            <div class="grid col-span-2 grid-rows-2 place-content-evenly place-items-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/rzeka.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
+                                </div>
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 delay-300 duration-1000 taos:scale-75 taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/energylandia.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:136]:
```suggestion
                                    <img src="{{ asset('images/photos/energylandia.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 5]
```diff
@@ -1,5 +1,172 @@
...
+                </div>
+            </div>
+            <div class="hidden xl:flex basis-2/3 gap-2 2xl:gap-6 justify-center">
+                <div class="grid grid-cols-1 place-items-center gap-2 2xl:gap-4">
+                    <div class="grid grid-rows-2 place-items-center gap-2 2xl:gap-6">
+                        <div class="grid grid-cols-3 place-content-evenly gap-2 2xl:gap-6">
+                            <div class="grid col-span-2 grid-rows-2 place-content-evenly place-items-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/rzeka.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:132]:
```suggestion
                                    <img src="{{ asset('images/photos/rzeka.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 6]
```diff
@@ -1,5 +1,172 @@
...
+                        {{__("content.about.section_4.subtitle_1")}}
+                        <span class="font-medium leading-relaxed">{{__("content.about.section_4.subtitle_2")}}</span>
+                        {{__("content.about.section_4.subtitle_3")}}
+                    </div>
+                </div>
+            </div>
+            <div class="flex xl:hidden">
+                <div class="size-full 2xl:size-52 3xl:size-72 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                     data-taos-offset="400">
+                    <img src="{{ asset('images/photos/lasertag.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl" alt="placeholder"/>
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:122]:
```suggestion
                    <img src="{{ asset('images/photos/lasertag.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl" alt="placeholder"/>
```

File: resources/views/about.blade.php [hunk 7]
```diff
@@ -1,5 +1,172 @@
...
+                </div>
+            </div>
+            <div class="hidden xl:flex basis-2/3 gap-2 2xl:gap-6 justify-center">
+                <div class="grid grid-cols-1 place-items-center gap-2 2xl:gap-4">
+                    <div class="grid grid-rows-2 place-items-center gap-2 2xl:gap-6">
+                        <div class="grid grid-cols-3 place-content-evenly gap-2 2xl:gap-6">
+                            <div class="grid col-span-2 grid-rows-2 place-content-evenly place-items-center gap-2 2xl:gap-6">
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:129]: So may `grid` class I see here. Can this be simplified?

File: resources/views/about.blade.php [hunk 8]
```diff
@@ -1,5 +1,172 @@
...
+                        <span class="font-medium leading-relaxed">{{__("content.about.section_3.content_5")}}</span>{{__("content.about.section_3.content_6")}}
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="bg-gradient-to-b from-gray-light to-white">
+        <x-waves.wave-2.top></x-waves.wave-2.top>
+        <div class="relative flex flex-col xl:flex-row text-center md:text-left px-[10%] md:px-[20%] gap-x-16 py-8 xl:py-28 z-20">
+            <div class="flex flex-col basis-1/3">
```
Comment (dawidrudnik) [inline resources/views/about.blade.php:104]: `basis-1/3` class can be problematic in mobile version.

Comment (AleksandraKozubal) [reply resources/views/about.blade.php]: switched to grid, right now grid cols 3 is used only on 2xl screens so it shouldn't be a problem

File: resources/views/about.blade.php (hunk 7 above)
Comment (AleksandraKozubal) [reply resources/views/about.blade.php]: yes, changed that

**Comment by AleksandraKozubal** [comment]
In the last commit I changed a few things after @bl4cksugar review.

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. I like the idea of language in dropdown, but this language list is too narrow and it's rounded, but I think it should be not rounded.

image

2. Really nice that we removed all animations, because now its a lot better. But when I refresh page I see that logo and CTA button on navigation are moving - can we fix this somehow?

https://github.com/blumilksoftware/website/assets/56546832/e40ada30-9aac-4bc9-89b8-785802c13ee2

3. I don't know why some images are not zoom a bit, when I hover mouse cursor
(and funfact - somehow now photos are not overlapping anymore)


https://github.com/blumilksoftware/website/assets/56546832/67eff131-ed9e-49b0-ad38-55404a0acfe5

**Comment by AleksandraKozubal** [comment]
> 1. I like the idea of language in dropdown, but this language list is too narrow and it's rounded, but I think it should be not rounded.
> 
>  2. Really nice that we removed all animations, because now its a lot better. But when I refresh page I see that logo and CTA button on navigation are moving - can we fix this somehow?
> 
> 3. I don't know why some images are not zoom a bit, when I hover mouse cursor
>        (and funfact - somehow now photos are not overlapping anymore)

1. changed, it is square as dropdown toggle
2. It should be better right now
3. fixed

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Line height for bold text is different than line height fot normal text

image

**Comment by AleksandraKozubal** [comment]
> Line height for bold text is different than line height fot normal text
> 
> image

fixed

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by dawidrudnik — APPROVED** [review]

**Pull request merged**